src/Microsoft.AspNetCore.Razor.Language/Intermediate/HtmlAttributeValueIRNode.cs

```diff
@@ -10,16 +10,14 @@ public sealed class HtmlAttributeValueIRNode : RazorIRNode
     {
         public override ItemCollection Annotations => ReadOnlyItemCollection.Empty;
 
-        public override IList<RazorIRNode> Children { get; } = EmptyArray;
+        public override IList<RazorIRNode> Children { get; } = new List<RazorIRNode>();
```

Comment:
We don't currently micro-manage things like this. Let's only do that once we have perf data to explainificate it